easybuild/easyblocks/p/python.py

```diff
@@ -144,10 +144,8 @@ def __init__(self, *args, **kwargs):
 
         self.pyshortver = '.'.join(self.version.split('.')[:2])
 
-        self.pythonpath = None
-        if self.cfg['ebpythonprefixes']:
-            easybuild_subdir = log_path()
-            self.pythonpath = os.path.join(easybuild_subdir, 'python')
+        # Used for EBPYTHONPREFIXES handler script
+        self.pythonpath = os.path.join(log_path(), 'python')
```

Comment:
@ocaisa Setting this always here (instead of conditionally None) provides most of backwards compat. However there is a minor change especially with checking (now consistently) for `self.cfg.get('ebpythonprefixes')`